app/src/main/assets/DOC_HTML/apps/D_NFET.md

```diff
@@ -1,19 +1,19 @@
 N-channel FET
 ---
 
-## Dependence of drain current on D-S voltage, and Gate Voltage
+### Dependence of drain current on D-S voltage, and Gate Voltage
 
 ![](file:///android_asset/DOC_HTML/apps/images/schematics/njfet.svg@100%|auto)
 
-## Dependence of drain current on D-S voltage, and Gate Voltage
+### Dependence of drain current on D-S voltage, and Gate Voltage
 
 * Set the gate voltage for various values between 0, and -2 V , and observe the effect on saturation current
 
 * PV1 is varied in steps, and for each step the current is calculated from the difference between voltages at PV1 and CH1, and the known value of the resistor(560 ohms)
 
 * Acquired plots can be selectively displayed or deleted.
 
-## screenshot
+### screenshot
```

Comment:
Please keep it constant across all experiment files. If its `###` or `####`, it should be same across all files like a pre-defined format.

Reply:
@NamanRawat1 Please check other headings too and ensure heading, sub-heading, etc. are same across all experiment files.

Reply:
ok, I will make same across all files